P5/Source/Specs/teidata.namespace.xml

```diff
@@ -35,7 +35,7 @@ See the file COPYING.txt for details
     indicare i nomi degli spazi in XML come stabilito dalle raccomandazioni tecniche del W3C per gli
     <ref target="http://www.w3.org/TR/1999/REC-xml-names-19990114/">spazi dei nomi in XML</ref>.</desc>
   <content>
-      <dataRef name="anyURI"/>
+      <dataRef restriction="\S*" name="anyURI"/>
```

Comment:
why `\S*` and not `\S+` here?

Reply:
Good question. Because the `@ns` of some things (in particular, `<attDef>`) is sometimes empty.
I would not mind solving that problem by requiring _something_ here (i.e., use `\S+`), and having the defintion of `attDef/@ns` be `"teidata.namespace.xml | ''"`.

Reply:
Also, the current datatype (xs:anyURI without any restriction) allows nil. Thus I am going to assert that this ticket should not require characters to occur in the value of teidata.namespace. If we think one or more should be required (i.e., that nil should not be allowed, and the need to express “no namespace” should be moved to the definition of the attribute), that’s another ticket.